h2o-r/h2o-DESCRIPTION.template

```diff
@@ -26,6 +26,7 @@ Imports: graphics,
 Suggests: ggplot2,
           mlbench,
           Matrix,
+          slam,
```

Comment:
You are using the slam package without checking if the package is installed. Your code seems to depend on that package. Therefore the package should not be listed in the Suggests, it should be in Imports instead.

This is maybe a controversial dependency. Users will need to install one more package.